<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HSV Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSlider" name="h_slider">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>179</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Hue (0-179)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="s_slider">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Saturation (0-255)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="v_slider">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Value (0-255)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>